discord-scripts/thread-management.ts

```diff
@@ -0,0 +1,134 @@
+import { Channel, ChannelType, Client } from "discord.js"
+
+// Emoji used to suggest a thread.
+const THREAD_EMOJI = "🧵"
+// Category that is treated as recreational, i.e. the rules don't apply baby.
+const RECREATIONAL_CATEGORY_ID = "1079492118692757605"
```

Comment:
I find it really helpful to set this kind of stuff as environment variables, that way I can have my local testbot run on a test server

<img width="1373" alt="image" src="https://user-images.githubusercontent.com/1045160/228556238-e9a0c9bf-bcd4-4a91-85e6-ceb7ea8142a5.png">